docs/release-notes.rst

```diff
@@ -5,6 +5,21 @@
 Release notes
 =============
 
+.. _release-1.0.5:
+
+itemloaders 1.0.5 (2022-08-NN)
+------------------------------
+
+-  Allow additional args to be passed when calling :meth:`ItemLoader.add_xpath` (:gh:`48`)
+
+-  Fixed missing space in exception message (:gh: `47`)
+
+-  Updated company name in author and copyright sections (:gh: `42`)
+
+-  Added official support for python 3.9 and improve PyPy compatibility (:gh: `44`)
+
+-  Added official support for python 3.10 (:gh: `53`)
```

Comment:
```suggestion
-  Allow additional args to be passed when calling :meth:`ItemLoader.add_xpath` (:gh:`48`)

-  Fixed missing space in an exception message (:gh:`47`)

-  Updated company name in author and copyright sections (:gh:`42`)

-  Added official support for Python 3.9 and improved PyPy compatibility (:gh:`44`)

-  Added official support for Python 3.10 (:gh:`53`)
```

Also, we could merge the last 2 bullet points into 1, and probably feature it first or second in the list.